pkg/cmd/cli/cli_changefeed_helper.go

```diff
@@ -44,7 +44,8 @@ func confirmLargeDataGap(cmd *cobra.Command, currentPhysical int64, startTs uint
 			return err
 		}
 		if strings.ToLower(strings.TrimSpace(yOrN)) != "y" {
-			return errors.NewNoStackError("abort changefeed create or resume")
+			cmd.Printf("abort changefeed create\n")
+			return cerror.ErrCliAborted.FastGenByArgs("cli changefeed create or resume")
```

Comment:
This message confused me.

Reply:
The above message said this is a `create`, but this message said this is a `create or resume`. Can we indicate it directly?

Reply:
both create and resume changefeed cli would call this func